Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ZoomableLineEdit.cpp

```diff
@@ -30,15 +30,15 @@ namespace EMStudio
         m_baseSize = sz;
     }
 
-    void ZoomableLineEdit::setBaseFontPointSizeF(qreal pointSize)
+    void ZoomableLineEdit::setBaseFontPixelSize(qreal pointSize)
     {
-        m_fontPointSize = pointSize;
+        m_fontPixelSize = pointSize;
     }
 
     void ZoomableLineEdit::updateScaledSize()
     {
         QFont f = font();
-        f.setPointSizeF(m_fontPointSize * m_scale);
```

Comment:
[uber nit] if it's convenient, now that you're making modifications around this code, please rename `f` variable to something more meaningful, like `font` or `qFont`.